src/toast-notification/classes.scss

```diff
@@ -60,13 +55,50 @@
 
     .iui-toast-entering,
     .iui-toast-entered {
-      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175) both;
+      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175)
+        both;
     }
 
     .iui-toast-exiting {
       animation: closeAnimation 120ms cubic-bezier(0.4, 0, 1, 1) both;
     }
   }
+
+  &.iui-placement-top {
+    margin-left: auto;
+    margin-right: auto;
+    left: 0;
+    top: 0;
+    right: 0;
+  }
+
+  &.iui-placement-top-start {
+    left: 0;
+    top: 0;
+  }
+
+  &.iui-placement-top-end {
+    right: 0;
+    top: 0;
+  }
+
+  &.iui-placement-bottom {
+    margin-left: auto;
+    margin-right: auto;
+    bottom: 0;
+    left: 0;
+    right: 0;
+  }
+
+  &.iui-placement-bottom-start {
+    left: 0;
+    bottom: 0;
+  }
+
+  &.iui-placement-bottom-end {
```

Comment:
Wonder if this is how it should look or if we want to push everything to the edge.

![](https://raw.githubusercontent.com/iTwin/iTwinUI/6b65b6dea5366b5cc73b1a13a9cd6592fba70c0b/backstop/results/bitmaps_reference/iTwinUI_dark_toast-notification_Bottom_end_placement_0_document_0_desktop.png)

Reply:
Added align-items: flex-start/flex-end; Now it looks much better